cmd/neofs-adm/internal/modules/morph/group.go

```diff
@@ -0,0 +1,113 @@
+package morph
+
+import (
+	"encoding/json"
+	"fmt"
+	"path"
+
+	"github.com/nspcc-dev/neo-go/cli/input"
+	"github.com/nspcc-dev/neo-go/pkg/crypto/keys"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract/manifest"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+	"github.com/nspcc-dev/neo-go/pkg/wallet"
+	"github.com/spf13/viper"
+)
+
+const contractWalletName = "contract.json"
+
+func initializeManifestWallet(walletDir string) error {
+	var (
+		password string
+		err      error
+	)
+
+	if key := "credentials.contract"; viper.IsSet(key) {
+		password = viper.GetString(key)
+	} else {
+		prompt := "Password for contract wallet > "
+		password, err = input.ReadPassword(prompt)
+		if err != nil {
+			return err
+		}
+	}
+
+	w, err := wallet.NewWallet(path.Join(walletDir, contractWalletName))
+	if err != nil {
+		return err
+	}
+
+	acc, err := wallet.NewAccount()
+	if err != nil {
+		return err
+	}
+
+	err = acc.Encrypt(password, keys.NEP2ScryptParams())
+	if err != nil {
+		return err
+	}
+
+	w.AddAccount(acc)
+	if err := w.Save(); err != nil {
+		return err
+	}
+
+	w.Close()
+	return nil
+}
+
+func openManifestWallet(walletDir string) (*wallet.Wallet, error) {
+	p := path.Join(walletDir, contractWalletName)
+	w, err := wallet.NewWalletFromFile(p)
+	if err != nil {
+		return nil, fmt.Errorf("can't open wallet: %w", err)
+	}
+
+	var password string
+	if key := "credentials.contract"; viper.IsSet(key) {
```

Comment:
Seems similar to code block in `initializeManifestWallet`, maybe unify them?
Btw `key` can be `const`.

Reply:
Fixed.